sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1336,6 +1336,22 @@ def _retry_zones(
         prev_cluster_status: Optional[status_lib.ClusterStatus],
     ):
         """The provision retry loop."""
+        # When rsync is not installed in the user's machine, Ray will
+        # silently retry to up the node for _MAX_RAY_UP_RETRY number of times
+        # This is time consuming so we fail early.
+        try:
+            subprocess.run('rsync --version',
+                           shell=True,
+                           check=True,
+                           stdout=subprocess.PIPE,
+                           stderr=subprocess.PIPE)
+        except subprocess.CalledProcessError as e:
+            with ux_utils.print_exception_no_traceback():
+                raise RuntimeError(
+                    'rsync is required for provisioning and it is not '
+                    'installed. Please run the following command:\n'
+                    '    $ sudo apt-get install rsync') from e
```

Comment:
`sudo apt-get install rsync` is system specific, i.e., only works for debian series of system. 
```suggestion
                    '`rsync` is required for provisioning and it is not '
                    'installed. For Debian/Ubuntu system, install it with:\n'
                    '  $ sudo apt install rsync') from e
```